databank/management/commands/sources/FDRS.py

```diff
@@ -127,12 +128,11 @@ def load(country, overview, fdrs_data):
         int(item["year"]) for item in fdrs_data.values() if item is not None and item.get("year") is not None
     )
 
+    # NOTE: We are getting the only latest year specific data
     for fdrs_indicator, field in FDRS_INDICATORS_FIELD_MAP:
-        value = fdrs_data.get(f"{country.iso.upper()}-{fdrs_indicator}")
-        setattr(
-            overview,
-            field.field.name,
-            value and value.get("value"),
-        )
+        data = fdrs_data.get(f"{country.iso.upper()}-{fdrs_indicator}")
+        if data and int(data.get("year")) == fdrs_data_fetched_year:
+            value = data.get("value")
+            setattr(overview, field.field.name, value)
```

Comment:
We will also need to clear the field's value if it doesn't exist in the remote source for the max year.